### PR TITLE
[Compute] CDRP SDK https://github.com/Azure/azure-rest-api-specs/pull/18351

### PR DESCRIPTION
**Pull request by haagha**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Comment by haagha** [comment]
@m-nash can you help with the CI failure? Also this is generated from preview Swagger, should we merge it to main?

File: sdk/compute/Microsoft.Azure.Management.Compute/src/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -7,9 +7,9 @@
 [assembly: AssemblyTitle("Microsoft Azure Compute Management Library")]
 [assembly: AssemblyDescription("Provides management functionality for Microsoft Azure Compute Resources.")]
 
-[assembly: AssemblyVersion("54.0.0")]
-[assembly: AssemblyFileVersion("54.0.0")]
-[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyVersion("54.1.0")]
```
Comment (archerzz) [inline sdk/compute/Microsoft.Azure.Management.Compute/src/Properties/AssemblyInfo.cs:10]: Keep the `AssemblyVersion`, unless it's a major change. See: https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/78/Mgmt-.NET-SDK-release-process?anchor=signing-and-publishing-nuget-packges

**Review by archerzz — CHANGES_REQUESTED** [review]
Need to change the API tag in https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/compute/Microsoft.Azure.Management.Compute/AzSdk.RP.props

**Comment by haagha** [comment]
> Need to change the API tag in https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/compute/Microsoft.Azure.Management.Compute/AzSdk.RP.props

@archerzz This is supposed to be preview SDK, if I change API tag it will show in consequent stable SDK's. Is that fine?
2nd question: When I publish the SDK should I do it from main or cdrpsdk branch?

**Comment by archerzz** [comment]
> > Need to change the API tag in https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/compute/Microsoft.Azure.Management.Compute/AzSdk.RP.props
> 
> @archerzz This is supposed to be preview SDK, if I change API tag it will show in consequent stable SDK's. Is that fine? 2nd question: When I publish the SDK should I do it from main or cdrpsdk branch?

@haagha It's OK to have preview api tag. You can see them in other SDKs as well, like [this](https://github.com/Azure/azure-sdk-for-net/blob/35eb8d55e6e37fb02b4422f3a65419b3ed3ac1cd/sdk/authorization/Microsoft.Azure.Management.Authorization/AzSdk.RP.props).
It's recommended to release from `main`. You can specify specific commit in the release job, if necessary.

**Review by archerzz — APPROVED** [review]

**Pull request merged**